eva_submission/eload_preparation.py

```diff
@@ -15,6 +15,9 @@
 from eva_submission.submission_in_ftp import FtpDepositBox
 from eva_submission.xlsx.xlsx_parser_eva import EvaXlsxReader, EvaXlsxWriter
 
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
```

Comment:
As `Eload` inherits from [`AppLogger`](https://github.com/EBIvariation/eva-common-pyutils/blob/master/ebi_eva_common_pyutils/logger.py#L123), you can leave this out and just use `self.warning` etc.

Reply:
updated